testclient/src/main/java/org/corfudb/testclient/SequencerFlood.java

```diff
@@ -0,0 +1,127 @@
+package org.corfudb.testclient;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import lombok.extern.slf4j.Slf4j;
+import org.HdrHistogram.Histogram;
+import org.HdrHistogram.Recorder;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.util.Sleep;
+
+import java.time.Duration;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.LongAdder;
+
+/**
+ * Created by Maithem on 8/1/19.
+ */
+@Slf4j
+public class SequencerFlood {
+
+    static class Args {
+        @Parameter(names = {"-h", "--help"}, description = "Help message", help = true)
+        boolean help;
+
+        @Parameter(names = {"--endpoint"}, description = "Cluster endpoint", required = true)
+        String url;
+
+        @Parameter(names = {"--num-clients"}, description = "Number of clients", required = true)
+        int numClients;
+
+        @Parameter(names = {"--num-threads"}, description = "Number of threads", required = true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651469)